include/wabt/shared-validator.h

```diff
@@ -289,6 +277,11 @@ class SharedValidator {
     Index end;
   };
 
+  struct LocalReferenceMap {
+    Type type;
+    Index bit_index;
```

Comment:
IIRC, this is an index into local_ref_is_set_?   Perhaps it should use be size_t rather than `Index`?

Perhaps it should be called `is_set_index`?  Or something like that?